index.html

```diff
@@ -817,18 +821,31 @@ <h3 id="requesting-more-permission">
           <li>If <var>current state</var> is not {{PermissionState/"prompt"}}, return <var>current
           state</var> and abort these steps.
           </li>
-          <li>Ask the user for <a>express permission</a> for the calling algorithm to use the
-          <a>powerful feature</a> described by |descriptor|.
+          <li>If the [=user agent=] has an [=administrator=] who has configured the [=user agent=]
+          to grant or deny permission for the calling algorithm to use the <a>powerful feature</a>
+          described by |descriptor|:
+          <ol>
+            <li>Set |current state| to {{PermissionState/"granted"}} if the [=administrator=]
+            granted permission; otherwise to {{PermissionState/"denied"}}.</li>
+            <li>Inform the user of the [=administrator=]'s configuration.</li>
```

Comment:
Similarly here, should we only inform the user if the admin configuration forced a 'grant' result?

Reply:
This seems to assume the policy is being applied in real time?